schemas/taip-0/DepositRequest.json

```diff
@@ -0,0 +1,66 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://tap.rspv/schemas/DepositRequest.json",
+  "title": "DepositRequest",
+  "description": "A deposit request",
+  "type": "object",
+  "required": [
+    "transactionId",
+    "amount",
+    "asset",
+    "originators",
+    "beneficiaries",
+    "agents",
+    "settlementInfo"
+  ],
+  "properties": {
```

Comment:
I've omitted the `originatorVP` and `beneficaryVP` from the proposal here in favor of some of the newer methods for exchanging PII e.g. [WACI](https://identity.foundation/waci-didcomm/v1.0/#step-3-send-message-requesting-presentation), [Selective Disclosure](https://datatracker.ietf.org/doc/html/draft-ietf-oauth-selective-disclosure-jwt-05)